src/main/kotlin/org/jqassistant/tooling/intellij/plugin/editor/report/ReportNode.kt

```diff
@@ -0,0 +1,105 @@
+package org.jqassistant.tooling.intellij.plugin.editor.report
+
+
+import org.jqassistant.schema.report.v2.ConstraintType
+import org.jqassistant.schema.report.v2.GroupType
+import org.jqassistant.schema.report.v2.ReferencableRuleType
+import org.jqassistant.schema.report.v2.RowType
+import java.util.*
+import javax.swing.tree.TreeNode
+import kotlin.collections.ArrayList
+
+
+/**
+ * Extension function for converting a {@link List} to an {@link Enumeration}
+ */
+fun <T> List<T>.toEnumeration(): Enumeration<T> {
```

Comment:
There is `Collections.enumeration` for this purpose. No need to implement a custom enumeration.